docs/content.zh/docs/connectors/datastream/dynamodb.md

```diff
@@ -0,0 +1,170 @@
+---
+title: DynamoDB
+weight: 5
+type: docs
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Amazon DynamoDB Sink
+
+The DynamoDB sink writes to [Amazon DynamoDB](https://aws.amazon.com/dynamodb) using the [AWS v2 SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html).
+
+Follow the instructions from the [Amazon DynamoDB Developer Guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/getting-started-step-1.html)
+to setup a table.
+
+To use the connector, add the following Maven dependency to your project:
+
+{{< artifact flink-connector-dynamodb >}}
```

Comment:
I wonder how this will work with different flink versions

Reply:
These docs are included into the Flink docs, so it should resolve the variables ok

Reply:
I meant more the connector - flink mapping!

e.g.
`1.0.0_1.15` or `1.1.0_1.15`. How will we know whether to include `1.1.0` or `1.0.0`?

Reply:
use the `connector_artifact` shortcode and supply the version of the connector _without_ the Flink version suffix.
The Flink docs then integrate the docs of the connector version that is compatible with Flink, and the connector_artifact shortcode injects the corresponding version suffix.